lib/src/modules/thumb/Thumb.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { node, string } from 'prop-types';
+import { useImage } from 'react-image';
```

Comment:
oooh what does this give us?

Reply:
Our main benefit is not managing the logic for everything load-related with images. E.g. `useImage` with an src gives us a loading state which we can then use in the UI 👍 (includes failures)